tmail-backend/blob/blobid-list/src/main/java/com/linagora/tmail/blob/blobid/list/BlobStoreConfigurationValidationStartUpCheck.java

```diff
@@ -0,0 +1,44 @@
+package com.linagora.tmail.blob.blobid.list;
+
+import javax.inject.Inject;
+
+import org.apache.james.lifecycle.api.StartUpCheck;
+import org.apache.james.modules.blobstore.validation.EventsourcingStorageStrategy;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public class BlobStoreConfigurationValidationStartUpCheck implements StartUpCheck {
```

Comment:
Do we really need to copy this too? Can't reuse the existing one by importing the module?

Reply:
Yes we need because the configuration POJO is different... For you defense I asked myself the same question.

Reply:
Ah the joys of Guice. Ok thanks for the answer :)